src/common/commonutils/CommonUtils.h

```diff
@@ -100,11 +100,11 @@ int FindTextInCommandOutput(const char* command, const char* text, char** reason
 int CheckLockoutForFailedPasswordAttempts(const char* fileName, void* log);
 int CheckOnlyApprovedMacAlgorithmsAreUsed(const char** macs, unsigned int numberOfMacs, char** reason, void* log);
 int CheckAppropriateCiphersForSsh(const char** ciphers, unsigned int numberOfCiphers, char** reason, void* log);
-int CheckLimitedUserAcccessForSsh(const char** values, unsigned int numberOfValues, char** reason, void* log);
-int CheckSshOptionIsSetToString(const char* option, const char* expectedValue, char** reason, void* log);
-int CheckSshOptionIsSetToInteger(const char* option, int expectedValue, int* actualValue, char** reason, void* log);
-int CheckSshIdleTimeoutInterval(char** reason, void* log);
+int CheckSshOptionIsSet(const char* option, const char* expectedValue, char** actualValue, char** reason, void* log);

```

Comment:
Should't it be `CheckSSHOptionIsSet`? Looks better that way and also properly pascal cased.

Reply:
3 letter acronyms are cased and is also the common pattern in this project.